### PR TITLE
feat(ci): ensure dandling previews are removed every day a 2 am

### PR DESCRIPTION
**Pull request by ValentinTrinque**

### CODE REVIEW

**Review by mattrussell36 — APPROVED** [review]

File: .github/workflows/cleanup-resources-daily.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    runs-on: ubuntu-22.04
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: 'eu-west-1'
+    steps:
+      - name: Delete ${{ matrix.app }} preview app
+        run: |
+          # Retrieve all preview buckets
+          aws s3api list-buckets | jq -c -r '[.Buckets[] | select(.Name | contains("previews.vega.rocks"))]' > all_buckets
```
Comment (mattrussell36) [inline .github/workflows/cleanup-resources-daily.yml:19]: Is jq going to be installed?

File: .github/workflows/cleanup-resources-daily.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+jobs:
+  cleanup-previews:
+    name: Clean up dandling previews
+    runs-on: ubuntu-22.04
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: 'eu-west-1'
+    steps:
+      - name: Delete ${{ matrix.app }} preview app
```
Comment (daniel1302) [inline .github/workflows/cleanup-resources-daily.yml:16]: I think @mattrussell36 is right. We should install aws-cli first

https://github.com/marketplace/actions/install-aws-cli-action

Comment (ValentinTrinque) [reply .github/workflows/cleanup-resources-daily.yml:16]: It's install by default on GH machines now.

File: .github/workflows/cleanup-resources-daily.yml (hunk 1 above)
Comment (ValentinTrinque) [reply .github/workflows/cleanup-resources-daily.yml:19]: Yes.

**Review by mattrussell36 — APPROVED** [review]

**Review by edd — APPROVED** [review]

**Pull request merged**